base_app/src/app/app.component.ts

```diff
@@ -769,7 +788,7 @@ export class AppComponent implements OnInit, AfterViewInit /*, OnChanges */ {
                 tab._pendingRequestExpiration = now.valueOf() + this._pendingRequestWait;
 
                 const ajaxRequest = {
-                    url: AppComponent.getDataTabRelativePath(tab.dataUrl, tab.index, tab_hash, serverSideJsDebugging),
+                    url: `http://localhost:4100${AppComponent.getDataTabRelativePath(tab.dataUrl, tab.index, tab_hash, serverSideJsDebugging)}`,
```

Comment:
Can't use 4100 for the port - must use the prop-defined value for simpleui-server.  Otherwise, this breaks apps that are using other port numbers. (Unless we've modified the dockerized version of simpleui to always use 4100.

Reply:
fixed with you last commit